doc/source/theory/dem/dem.rst

```diff
@@ -112,7 +112,8 @@ The parameters are computed as followed:
     \frac{1}{Y_{e}} &= \frac{\left(1-\nu_i^2\right)}{Y_i}+\frac{\left(1-\nu_j^2\right)}{Y_j} \\
     \beta &= \frac{\ln{e}}{\sqrt{\ln^2{e}+\pi^2}} \\
     S_n &= 2Y_{e}\sqrt{R_{e}\delta_n} \\
-    S_t &= 8G_{e}\sqrt{R_{e}\delta_n}
+    S_t &= 8G_{e}\sqrt{R_{e}\delta_n} \\
+    \nu_e &= \frac{\nu_i, \nu_j}{2}
```

Comment:
```suggestion
    \nu_e &= \frac{\nu_i + \nu_j}{2}
```

Reply:
I would put is under 1/Y_e, also, don't you wanna use harmonic mean?

Reply:
Removed